src/lwc/utilInput/utilInput.js

```diff
@@ -70,23 +73,27 @@ export default class utilInput extends LightningElement {
     }
 
     get checkboxDefaultValue() {
-        return (this.defaultValue === TRUE || this.defaultValue === true) ? true : false;
+        return (this.defaultValue === TRUE || this.defaultValue === true);
     }
 
     get isWidget() {
-        return this.formFieldType === WIDGET ? true : false;
+        return this.formFieldType === WIDGET;
+    }
+
+    get isTrueFalsePicklist() {
+        return this.type === 'BOOLEAN' && this.sourceFieldDescribe && this.sourceFieldDescribe.dataType === 'Picklist';
```

Comment:
WE have the const BOOLEAN_TYPE = 'BOOLEAN' defined in a few files, could we consolidate and reference it here?